### ISSUE TITLE
Drag 'n Drop allowed by Place Marker/Replace with Other select marker function and does not end well

### ISSUE THREAD

**Issue by riverwanderer** [vassalengine/vassal#11451]
To replicate:  Take a Place Marker trait, use the Select option to navigate to a piece. Try moving the piece in the window into some other part of the piece palette (another folder ideally).  The piece may disappear and won't re-appear if you save and re-open the module.

Vassal v3.6.6.

**Comment by Cattlesquat** [comment]
I can't quite understand what to do to repro this? "use the select option"? "navigate to the piece?" "in the window"? I also can't quite tell if I'm drag-and-dropping an editor component? Or from a piece palette like during the game? Or some combo of both?

Maybe some screenshots would help?

**Comment by Cattlesquat** [comment]
@riverwanderer let me know if you have any screenshots / additional info for this

**Comment by assaultdoor** [comment]
I've noticed the same thing. From Place Marker, pick "Select".
<img width="382" alt="image" src="https://user-images.githubusercontent.com/61174371/219967512-ed7a93a4-68c9-41ce-a2fa-f2a2deb16fc6.png">
I grab test object 1 out of the test 1 folder and drag it into the test 2 folder.
<img width="267" alt="image" src="https://user-images.githubusercontent.com/61174371/219967561-3a56bb42-aad2-4736-8583-d707664b8423.png">
I drop it in there, and now it's just gone. The test 1 folder is empty, and the test 2 folder is unchanged.
<img width="267" alt="image" src="https://user-images.githubusercontent.com/61174371/219967623-9bd2da2d-98e0-4360-a657-d256df6c68ba.png">
I canceled out of all the windows and got back to the main editor interface, so no changes should have been made. I can still see test piece 1 in the test 1 folder the Game Piece Palette. However, when I try adding a Place Marker trait again and press Select, test object 1 is still gone.
